docs/usage/shoot_maintenance.md

```diff
@@ -1,3 +1,7 @@
+---
+description: Defining the maintenance time window, configuring automatic version updates, confine reconcilations only to happen during maintenance, adding an additional maintenance operation and list of special operations during maintenance
```

Comment:
I feel that this description is too long. Could it be shortened?

Reply:
I think that `list of special operations during maintenance` can be removed because it does not provide actionable information to the user

Reply:
I've added a new comment to reflect this change